single-work.php

```diff
@@ -0,0 +1,74 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
```

Comment:
single work shouldnt be in a `#posts` section

Reply:
what should it be in

Reply:
you know i've never actually used that ID for anything

Reply:
i personally think it should be either the wordpress name for the template aka `#single-work` or acceptably if the convention is here `#posttype-puralornah` aka `#work-single` or probably better just totally delete the whole element if it isn't needed

Reply:
seems like the wrong use for the `<section>` elem

Reply:
to me a `<section>` is more like the whole gallery. which has multiple rows, etc

Reply:
i'll make an issue for this to do theme-wide